dashboard/app/models/level.rb

```diff
@@ -186,6 +186,20 @@ def self.key_to_params(key)
     end
   end
 
+  # Returns true if this is a pixelation level.
+  def pixelation?
+    # TODO(dave|joshlory): Define Pixelation < DSLDefined as a new level type,
+    # eliminating this fragile test of 'href'.
+    self.is_a?(DSLDefined) && self.try(:properties).try(:[], "href") == "pixelation/pixelation.html"
```

Comment:
No specific situation that I'm aware of, but since this logic is specific to pixelation, I didn't want it to choke if the implementation of some other level type changed.